playbooks/installation/index.adoc

```diff
@@ -95,6 +95,11 @@ Each service in the cluster will be assigned an IP from this range.  The default
 
 Both the openshift_portal_net and the osm_cluster_network_cidr will default to the value above if not set, however if you prefer to have your inventory file describe your environment, its recommended to set them explicitly for future reference or comparison.
 
+==== Host Subnet Length
+*Ansible Host File Variable:* `osm_host_subnet_length=9`
```

Comment:
If we're going to list the variable, we should also add it to our host file samples.